internal/ecsservicediscovery/processorstats.go

```diff
@@ -43,11 +43,7 @@ func (sd *ProcessorStats) AddStatsCount(name string, count int) {
 	if sd.stats == nil {
 		sd.ResetStats()
 	}
-	if _, ok := sd.stats[name]; ok {
-		sd.stats[name] += count
-	} else {
-		sd.stats[name] = count
-	}
+	sd.stats[name] += count
```

Comment:
unrelated go lint change.
Not *having* to check if a map contains a key is a nice feature of golang.